frontend/src/types/CSVLog.ts

```diff
@@ -0,0 +1,10 @@
+export type CSVLog = {
+    attnTo: string;
+    building: string;
+    datetime: string;
+    employee: string;
+    flagged: boolean;
+    note: string;
+    residentId: string;
+    tags: string;
+};
```

Comment:
nit: I think we might want to run the linter on this 🤓🤓 (we use default ESLint settings, and I think it enforces empty lines in files)